src/pages/Blog.js

```diff
@@ -0,0 +1,7 @@
+const Blog = () => {
+	return `
+		<h2>Blog</h2>
+	`;
```

Comment:
It's better if you create the template in a variable and then return it like:

`
const Blog = () =>
    const view = ... //your template
    return view
`
Do it for the files that contains the same structure.

Reply:
Read my comment